pandas/tests/apply/test_frame_apply.py

```diff
@@ -1451,7 +1451,10 @@ def func(row):
     result = df.apply(func, axis=1)
 
     tm.assert_frame_equal(result, expected)
-    tm.assert_frame_equal(df, result)
+    if not using_array_manager:
+        # INFO(ArrayManager) With BlockManager, the row is a view and mutated in place,
+        # with ArrayManager the row is not a view, and thus not mutated in place
```

Comment:
is there anything we should be asserting in the AM case?

Reply:
Added the equivalent assert (but so asserting it *didn't* change) for AM